tests/functional/simple_copy/test_copy_uppercase.py

```diff
@@ -0,0 +1,66 @@
+import pytest
+import os
+from tests.tables import TableComparison
+from tests.util import run_dbt
+
+from tests.functional.simple_copy.test_simple_copy import (
```

Comment:
This makes me think: What would it look like if an external plugin wanted to run `test_simple_copy`, exactly as written, with no changes? What would it look like if they wanted to run it, but needed to override a few of the fixtures?

Reply:
I've been looking at options for sharing the fixtures, and what I found was using them as pytest plugins. I'm not fully up to speed on how that would work, but I think we would include the base conftest.py from the dbt-core repo (or a test fixture repo?), and the provide fixture overrides in a conftest.py in the adapter repos. Or something like that.